sdk/core/core-client-rest/test/browser/streams.spec.ts

```diff
@@ -29,43 +28,51 @@ function createResponse(statusCode: number, body = ""): Response {
 const mockBaseUrl = "https://example.org";
 
 describe("[Browser] Streams", () => {
-  let fetchMock: sinon.SinonStub;
   beforeEach(() => {
-    fetchMock = sinon.stub(self, "fetch");
+    vi.stubGlobal("fetch", vi.fn());
   });
 
   afterEach(() => {
-    sinon.restore();
+    vi.clearAllMocks();
```

Comment:
well I guess its explicit 😄

Reply:
Should be `vi.restoreAllMocks`.  Fixing now.